Test/NUnit3.Tests.ps1

```diff
@@ -391,29 +448,37 @@ function ThenRanOnlySpecificTest
     }
 }
 
+function GetGeneratedNUnitReport {

```

Comment:
Move this to the top with the other functions, ordered alphabetically. Rename to `Get-GeneratedNUnitReport1`.